tests/ops/integration_tests/test_integration_erasure_order.py

```diff
@@ -0,0 +1,350 @@
+import random
+from typing import Any, Dict, List
+from unittest import mock
+
+import pytest
+from sqlalchemy.orm import Session
+
+from fides.api.ops.common_exceptions import TraversalError
+from fides.api.ops.graph.graph import DatasetGraph
+from fides.api.ops.graph.traversal import TraversalNode
+from fides.api.ops.models.policy import ActionType, Policy
+from fides.api.ops.models.privacy_request import ExecutionLog, PrivacyRequest
+from fides.api.ops.schemas.redis_cache import Identity
+from fides.api.ops.service.connectors.saas.authenticated_client import (
+    AuthenticatedClient,
+)
+from fides.api.ops.service.saas_request.saas_request_override_factory import (
+    SaaSRequestType,
+    register,
+)
+from fides.api.ops.task import graph_task
+from fides.api.ops.task.graph_task import get_cached_data_for_erasures
+from fides.api.ops.util.collection_util import Row
+from fides.core.config import get_config
+from tests.ops.graph.graph_test_util import assert_rows_match
+
+CONFIG = get_config()
+
+
+def erasure_execution_logs(
+    db: Session, privacy_request: PrivacyRequest
+) -> List[ExecutionLog]:
+    """Returns the erasure execution logs for the given privacy request ordered by created_at"""
+    return (
+        db.query(ExecutionLog)
+        .filter_by(
+            privacy_request_id=privacy_request.id, action_type=ActionType.erasure
+        )
+        .order_by("created_at")
+        .all()
+    )
+
+
+# custom override functions to facilitate testing
+@register("read_no_op", [SaaSRequestType.READ])
+def read_no_op(
+    client: AuthenticatedClient,
+    node: TraversalNode,
+    policy: Policy,
+    privacy_request: PrivacyRequest,
+    input_data: Dict[str, List[Any]],
+    secrets: Dict[str, Any],
+) -> List[Row]:
+    return [{f"{node.address.collection}_id": 1}]
+
+
+@register("delete_no_op", [SaaSRequestType.DELETE])
+def delete_no_op(
```

Comment:
Moved these no-ops into the test to avoid having them available for general use like the other custom override functions